server/controllers/ActivityType.controller.js

```diff
@@ -0,0 +1,61 @@
+///////////////////////////// Import dependencies ///////////////////////////////
+const ActivityType = require('../models/ActivityType.model');
+/////////////////////////////////////////////////////////////////////////////////
+
+
+/////////////////////////////////////////// CRUD Callbacks for Activity Types ///////////////////////////////////////////
+exports.getAllActivityTypes = (req, res) => {
```

Comment:
can be rename to just getAll since route is unique (/activity_types/getAll)

Reply:
Done